src/index.ts

```diff
@@ -11,11 +11,11 @@ async function openDiffWindow(noteIds : string[]) {
 	}
 	const noteId = noteIds[0];
 
-	try {
+	//try {
 		await diffWindow.OpenWindow(noteId);
-	} catch(ex) {
-		joplin.views.dialogs.showMessageBox(ex.message);
-	}
+	//} catch(ex) {
+	//	joplin.views.dialogs.showMessageBox(ex.message);
+	//}
```

Comment:
Can this code be removed? Or should it be uncommented?

Reply:
Oh yeah it indeed was a mistake haha. Uncommented now. I'm planning down the road to create custom Error types so that I can only show the ones the user should see in message boxes while anything else unexpected can go into console.log.